### PR TITLE
Changed distance/duration parms to null due to change at server end.

### LINKED ISSUE #466: Circular routes has stopped working
https://www.cyclestreets.net/api/journey.json?plan=leisure&itinerarypoints=0.03201484680175781%2C51.46852571389458&distance=25000&duration=0&poitypes=atms&reporterrors=1&key=...

gives error message "Parameter duration is not a naturalNumber."

Removing the duration parameter returns the route OK:
https://www.cyclestreets.net/api/journey.json?plan=leisure&itinerarypoints=0.03201484680175781%2C51.46852571389458&distance=25000&poitypes=atms&reporterrors=1&key=...

Has something changed in the API?

### PR DESCRIPTION
**Pull request by HilaryN**
To fix #466 (circular routes no longer working)

### CODE REVIEW

**Review by jezhiggins — APPROVED** [review]

File: libraries/cyclestreets-fragments/src/main/java/net/cyclestreets/RouteMapFragment.kt [hunk 1]
```diff
@@ -105,9 +105,11 @@ class RouteMapFragment : CycleMapFragment(), Route.Listener {
         super.onActivityResult(requestCode, resultCode, data)
         if ((requestCode == CIRCULAR_ROUTE_ACTIVITY_REQUEST_CODE) && (resultCode == Activity.RESULT_OK)) {
             if (data != null) {
+                val distance = data.getIntExtra(EXTRA_CIRCULAR_ROUTE_DISTANCE, 0)
+                val duration = data.getIntExtra(EXTRA_CIRCULAR_ROUTE_DURATION, 0)
```
Comment (oliverlockwood) [inline libraries/cyclestreets-fragments/src/main/java/net/cyclestreets/RouteMapFragment.kt:109]: How annoying that this only allows primitives, so you have to do the second defaulting below!

Ah well.

**Review by oliverlockwood — APPROVED** [review]

**Comment by oliverlockwood** [comment]
Thanks for fixing promptly, Hilary.

File: libraries/cyclestreets-fragments/src/main/java/net/cyclestreets/RouteMapFragment.kt (hunk 1 above)
Comment (HilaryN) [reply libraries/cyclestreets-fragments/src/main/java/net/cyclestreets/RouteMapFragment.kt:109]: I'm still getting this error message on my phone, which is on beta version.  When should I expect the changes to take effect?

**Comment by HilaryN** [comment]
I'm still getting this error message on my phone, which is on beta version.  When should I expect the changes to take effect?

**Comment by oliverlockwood** [comment]
Looks like the master build failed:
https://travis-ci.com/github/cyclestreets

I'll investigate why.

No build, no update on the play store!

**Comment by oliverlockwood** [comment]
@HilaryN after my Gradle changes the master build has just passed.  So at some point today the updated version - includeing this fix of yours - should become available on the Play store to beta users.

**Pull request merged**